hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/LedgerConfig.java

```diff
@@ -29,9 +29,9 @@
 @ConfigData("ledger")
 public record LedgerConfig(
         @ConfigProperty(defaultValue = "5000") @NetworkProperty int maxAutoAssociations,
-        @ConfigProperty(defaultValue = "100") @NetworkProperty int numSystemAccounts,
-        @ConfigProperty(defaultValue = "5000000000000000000") @Min(0) @NetworkProperty long totalTinyBarFloat,
-        @ConfigProperty(defaultValue = "0x03") @NetworkProperty Bytes id,
+        @ConfigProperty(defaultValue = "100") int numSystemAccounts,
+        @ConfigProperty(defaultValue = "5000000000000000000") @Min(0) long totalTinyBarFloat,
+        @ConfigProperty(defaultValue = "0x03") Bytes id,
```

Comment:
These values can all be different between nodes?
At minimum I would expect that system and float have to be the same across the network...

Reply:
Ya, that was the thing I was wrestling with. They cannot be separate, but they also cannot be overridden after the server starts. Kind of an immutable network property?

Reply:
Immutable network property makes sense (and we probably need a category for those).  Probably it should be possible to change under some circumstances (not for the Hedera network, but other network(s) using the same software will definitely want that capability).
Something to revisit in future work.